### PR TITLE
Change Cookie::getExpiresTimestamp() return type

### PR DESCRIPTION
**Pull request by cleptric**
see #12933

### CODE REVIEW

File: src/Http/Cookie/CookieInterface.php [hunk 1]
```diff
@@ -114,12 +114,12 @@ public function getExpiry();
     /**
      * Get the timestamp from the expiration time
      *
-     * Timestamps are strings as large timestamps can overflow MAX_INT
+     * Timestamps are integer as large timestamps can overflow MAX_INT
```
Comment (ADmad) [inline src/Http/Cookie/CookieInterface.php]: This statement needs to be removed, it makes no sense now :).

File: src/Http/Cookie/CookieInterface.php [hunk 2]
```diff
@@ -114,12 +114,9 @@ public function getExpiry();
     /**
      * Get the timestamp from the expiration time
      *
-     * Timestamps are strings as large timestamps can overflow MAX_INT
-     * in 32bit systems.
-     *
-     * @return string|null The expiry time as a string timestamp.
+     * @return int|null The expiry time as a string timestamp.
```
Comment (ADmad) [inline src/Http/Cookie/CookieInterface.php]: s/string/

Comment (cleptric) [reply src/Http/Cookie/CookieInterface.php]: `The expiry time as an integer timestamp.` ?

Comment (markstory) [reply src/Http/Cookie/CookieInterface.php]: Or 'expiry time as an integer'

File: src/Http/Cookie/CookieInterface.php [hunk 3]
```diff
@@ -114,7 +114,7 @@ public function getExpiry();
     /**
      * Get the timestamp from the expiration time
      *
-     * @return int|null The expiry time as a string timestamp.
+     * @return int|null The expiry time as an integer.
```
Comment (dereuromark) [inline src/Http/Cookie/CookieInterface.php]: How can this be null? I understand that 0 exists as unlimited, but what is null?

Comment (ADmad) [reply src/Http/Cookie/CookieInterface.php]: It's probably null because setcookie() allows the $expires argument to be null.

**Review by ADmad — APPROVED** [review]

**Pull request merged**